test/specs/reference/tests/06.permissions.js

```diff
@@ -1,8 +1,44 @@
-var nock = require("nock");
+var ermrestImport = require(process.env.PWD + '/test/utils/ermrest-import.js');
+
+exports.execute = (options) => {
+
+    var setCatalogAcls = (done, uri, catalogId, acls, cb, userCookie) => {
+        ermrestImport.importAcls(acls).then(() => {
+            removeCachedCatalog(catalogId);
+            if (userCookie) {
+                options.ermRest.setUserCookie(userCookie);
+            } else {
+                options.ermRest.resetUserCookie();
+            }
+            return options.ermRest.resolve(uri, { cid: "test" });
+        }).then((response) => {
+            cb(response);
+            done();
+        }, (err) => {
+            console.log(err);
+            done.fail();
+        });
+    }
+
+    var resetCatalogAcls = (done, acls) => {
+        ermrestImport.importAcls(acls).then(() => {
+            done();
+        }, (err) => {
+            console.log(err);
+            done.fail();
+        });
+    };
+
+    var removeCachedCatalog = (catalogId) => {
+        var server = options.ermRest.ermrestFactory.getServer(process.env.ERMREST_URL);
+        delete server.catalogs._catalogs[catalogId];
+    };
+
+    describe("For determining reference object permissions,", () => {
 
-exports.execute = function (options) {
+        var restrictedUserId = process.env.RESTRICTED_AUTH_COOKIE_ID,
```

Comment:
Please comment that `RESTRICTED_AUTH_COOKIE_ID` is something that the test framework is generating (I was confused at first that I might have to set it up myself).

Reply:
Yes, I will add the comment.